src/web/crate_details.rs

```diff
@@ -44,6 +44,7 @@ pub struct CrateDetails {
     pub(crate) doc_targets: Vec<String>,
     license: Option<String>,
     documentation_url: Option<String>,
+    doc_coverage: Option<f32>,
```

Comment:
This could actually be calculated on the template side of things from `total_items` and `documented_items` so we could not only have the coverage percentage but also a `100 out of 200 items documented` line in the coverage display

Reply:
Sure!